foreman/data_refinery_foreman/settings.py

```diff
@@ -93,6 +93,10 @@ def get_env_variable(var_name):
         'PASSWORD': get_env_variable('DATABASE_PASSWORD'),
         'HOST': get_env_variable('DATABASE_HOST'),
         'PORT': get_env_variable('DATABASE_PORT'),
+
+        'OPTIONS': {
+            'connect_timeout': get_env_variable('DATABASE_TIMEOUT')
```

Comment:
Out of curiosity: why did you need to adjust the timeout here? It's unusual to modify that.